packages/npm/@amazeelabs/graphql-directives/src/resolvers.ts

```diff
@@ -5,6 +5,12 @@ import { flow, isString } from 'lodash-es';
 export function createResolveConfig(
   schema: GraphQLSchema,
   directives: Record<string, GraphQLFieldResolver<any, any>>,
+  gatsbyHelpers?: {
```

Comment:
A nitpick: I would just call this `helpers`, `framework` or `api` and make it completely untyped. There is no benefit in knowing _what_ we pass through here, so we can as well keep it framework-independent.

Reply:
Done, renamed to helpers.

Reply:
actually, I renamed them to api now